### PR TITLE
Return zero when input version is same as target

### PR DESCRIPTION
**Pull request by reasonerjt**

### CODE REVIEW

File: tools/migration/cfg/migrate.py [hunk 1]
```diff
@@ -24,9 +25,12 @@ def main():
     input_version =  utils.get_conf_version(args.input_path)
     curr_dir = os.path.dirname(__file__)
     chain = []
+    if input_version == target_version:
+        print ("version of input cfg is identical to target:", input_version)
```
Comment (jessehu) [inline tools/migration/cfg/migrate.py]: suggestion: "The target version and source version in harbor.cfg  is the same: %s. No need to upgrade.", input_version

File: tools/migration/cfg/migrate.py [hunk 2]
```diff
@@ -24,9 +25,12 @@ def main():
     input_version =  utils.get_conf_version(args.input_path)
```
Comment (jessehu) [inline tools/migration/cfg/migrate.py]: 1) input_version could be 'unsupported', but seems we don't handle this case here.
2) Better to ensure target_version > input_version to do migration.
3) The logic in get_conf_version() is a little cool. Can we check the greatest tag of a harbor container image (e.g. harbor-log) to get current installed version?

Comment (reasonerjt) [reply tools/migration/cfg/migrate.py]: for 1) and 2) currently it's all handled in `search`, which tries to find a handler chain to migrate the .cfg file from source to target.  So both cases will fail as expected, if you are suggesting refining error messages, those are cosmetic change can be made in future.

for 3) For integration with VIC appliance, this script has to be executed in a container to which the harbor.cfg is single source of truth.

Comment (jessehu) [reply tools/migration/cfg/migrate.py]: Got it. Can we handle "input_version == target_version" in search too ?

Comment (jessehu) [reply tools/migration/cfg/migrate.py]: I think a message is needed to indicate why upgrade is not needed, at least print out the source version and target version.

File: tools/migration/cfg/migrate.py (hunk 1 above)
Comment (reasonerjt) [reply tools/migration/cfg/migrate.py]: Will remove this since we'll handle the `==` case in `search`

File: tools/migration/cfg/migrate.py (hunk 2 above)
Comment (reasonerjt) [reply tools/migration/cfg/migrate.py]: So then let's keep it as is for now.

**Pull request merged**